app/Services/UserService.php

```diff
@@ -52,7 +51,6 @@ public function delete($id)
     }
     public function search(Request $request)
     {
-        // $choose = $request->all();
         if($request->get('name') && $request->get('email'))
```

Comment:
@IDS-V-Tuanlt function search gì kỳ vậy, em test thử chưa, trường hợp người dùng nhập 1 trong 2 điều kiện thì sao?

Reply:
checked (The results returned as expected)